Jenkinsfile

```diff
@@ -1,12 +1,52 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+stage('Trust') {
+    enforceTrustedApproval('aerogear')
+}
+
+def prLabels = getPullRequestLabels {}
+
 def androidSdkSlave = containerTemplate(
   name: 'jnlp', 
   image: 'docker.io/aerogear/digger-android-slave-image:latest',
   args: '${computer.jnlpmac} ${computer.name}',
-  ttyEnabled: false) 
+  ttyEnabled: false)
+
+def runIntegrationTest() {
+  def repositoryName = "android-sdk"
+  def projectName = "test-${repositoryName}-${currentBuild.number}-${currentBuild.startTimeInMillis}"
+  def metricsApiTemplatePath = "./ci/aerogear-app-metrics.yml"
+  
+    openshift.withCluster() {
```

Comment:
Does it deploy to Wendy's OpenShift or somewhere else?

Reply:
Currently yes. We will probably want to make a use of the slaves behind VPN later.
Related ticket: https://issues.jboss.org/browse/AEROGEAR-2292